src/Scheduler.java

```diff
@@ -1,4 +1,94 @@
+import java.awt.Event;
+import java.util.ArrayList;
 
 public class Scheduler {
-
+	private ArrayList<Event> events;
+	private int floor;
+	// private static final NUM_FLOORS = 3
+	// might need an object to hold the event to give to elevator and wait() if null
```

Comment:
can you remove this file from your or, it's outdated and doesn't need to go into master